Sources/MapboxDirections/RouteRoadClassesViolations.swift

```diff
@@ -0,0 +1,69 @@
+
+import Foundation
+
+/**
+ Description of `RoadClasses` that were meant to be avoided during routing but still got used in a `Route`.
+ 
+ Such ignoring may be done by the engine if it cannot provide a route otherwise.
```

Comment:
I suggest to expand this comment a bit more. Questions raised in my head when I read this:
- What is the engine?
- Such ignoring? There is no mention of the ignoring in the doc comment except one this place. 
- "if it cannot provide a route otherwise", very hard to read. Maybe "otherwise" is a superfluous word. @1ec5 is it just me?

Reply:
If something in `roadClassesToAvoid` prevents the origin or destination from connecting to the main road network, the Directions API will temporarily ignore the road class constraints in order to reach the main road network.

Reply:
thanks for suggestion!